docs/sphinx/user_guide/components_migration.rst

```diff
@@ -0,0 +1,383 @@
+Migrating to GitLab CI Components
+===================================
+
+Starting with v2025.11.0, RADIUSS Shared CI provides GitLab CI Components as the
```

Comment:
Maybe check what release you want mentioned here and in the the components examples below